Tests/TuistKitTests/Services/FetchServiceTests.swift

```diff
@@ -192,4 +192,118 @@ final class FetchServiceTests: TuistUnitTestCase {
 
         XCTAssertFalse(dependenciesController.invokedUpdate)
     }
+
+    func test_fetch_path_is_found_in_root() async throws {
+        // Given
+        let stubbedPath = try temporaryPath()
+        let expectedFoundDependenciesLocation = stubbedPath.appending(
+            components: Constants.tuistDirectoryName, Manifest.dependencies.fileName(stubbedPath)
+        )
+        let stubbedDependencies = Dependencies(
+            carthage: nil,
+            swiftPackageManager: nil,
+            platforms: [.iOS, .macOS]
+        )
+        dependenciesModelLoader.loadDependenciesStub = { path, _ in
+            XCTAssertEqual(stubbedPath, path)
+            return stubbedDependencies
+        }
+
+        // Dependencies.swift in root
+        try fileHandler.touch(expectedFoundDependenciesLocation)
+
+        // When
+        try await subject.run(
+            path: stubbedPath.pathString,
+            update: false
+        )
```

Comment:
What are we asserting here? I'd recommend being more explicit about what we expect, either through the test code or documentation comments. For example, if what we are expecting here is that the function succeeds, let's assert that it doesn't throw an error or add a comment saying that the function invocation should succeed.

Reply:
This comment applies to all the test functions in this class.

Reply:
Thanks for this comment, and good callout, I forgot to add expectation.
The assertion takes place in the `loadDependenciesStub` closure.

I have added comments, and the expectation - please let me know if that is ok